subgraph/src/datasources/FortaStaking.ts

```diff
@@ -0,0 +1,248 @@
+import { Address, BigInt, ethereum } from "@graphprotocol/graph-ts";
+import { logStore, newMockEvent } from "matchstick-as";
+import {
+  StakeDeposited as StakeDepositedEvent,
+  Rewarded as RewardedEvent,
+  Slashed as SlashedEvent,
+  Froze as FrozeEvent,
+  FortaStaking as FortaStakingContract,
+} from "../../generated/FortaStaking/FortaStaking";
+import { Reward, Slash, Stake, Staker, Subject } from "../../generated/schema";
+import { fetchAccount } from "../fetch/account";
+
+import eventId from "../utils/event";
+
+export function handleStakeDeposited(event: StakeDepositedEvent): void {
```

Comment:
You don't want to track the event in the subgraph ???

Reply:
Can you elaborate further on this one?